### PR TITLE
[Feature:TAGrading] Auto open Images

### PR DESCRIPTION
**Pull request by ANS1514**
### What is the current behavior?

Images don't auto open in the TA grading panel

### What is the new behavior?
Images will auto open in the TA grading panel and will stay open even if you refresh the page.

### Other information?

I made a course on my local machine and created a discussion forum thread and made a post with an attachment. Then I made a homework linked to that thread, made a submission then went to grade it and use the discussion forum panel in the TA grader to see if the image opened.

### CODE REVIEW

**Review by cjreed121 — CHANGES_REQUESTED** [review]
Do you think you could instead put this logic into [DiscussionForumPanel.twig](https://github.com/Submitty/Submitty/blob/main/site/app/templates/grading/electronic/DiscussionForumPanel.twig)? That is much more closely tied to where we want this to happen. This would add a lot of extra computation on the forum since it would never apply.

File: site/app/templates/forum/CreatePost.twig [hunk 1]
```diff
@@ -108,6 +108,7 @@
 
 
         <div id="{{ post_attachment.params.well_id }}" class="attachment-well well well-lg"></div>
+
```
Comment (williamjallen) [inline site/app/templates/forum/CreatePost.twig:111]: This change seems to be unrelated and should be reverted

File: site/app/templates/grading/electronic/DiscussionForumPanel.twig [hunk 2]
```diff
@@ -9,11 +9,15 @@
...
-		});
+$('#posts_list').css('min-width', '-webkit-fill-available');
+$(function() {
+	generateCodeMirrorBlocks(document);
+});
+
+$(document).ready(function() {
+    loadAllInlineImages(true);
+
+});
```
Comment (williamjallen) [inline site/app/templates/grading/electronic/DiscussionForumPanel.twig:21]: All of this code should probably be moved to the relevant `.js` file.  Perhaps `ta-grading.js`.

**Review by williamjallen — CHANGES_REQUESTED** [review]

File: site/public/js/forum.js [hunk 3]
```diff
@@ -2501,3 +2501,4 @@ function pinAnnouncement(thread_id, type, csrf_token) {
         });
     }
 }
+
```
Comment (williamjallen) [inline site/public/js/forum.js:2504]: This line is unrelated to the rest of the changes and should be reverted.

**Review by williamjallen — CHANGES_REQUESTED** [review]
One quick change to prevent potential merge conflicts with other PRs and I'll approve.

**Review by williamjallen — APPROVED** [review]

**Pull request merged**